tests/shipperCustomer.js

```diff
@@ -1,36 +1,34 @@
-  await page.goto('https://milevision-stage.milezero.com/');
+  await page.goto('https://milevision-stage.milezero.com/#access_token=eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIiwiaXNzIjoiaHR0cHM6Ly9taWxlemVyby1zdGFnZS5hdXRoMC5jb20vIn0..C7ghf_EoFMX6B0up.wzvg6qqmpspdOlUT0HJzVCh69GzTrosn8vH8J432F95UhrRPQV4iIRoftsL8f6sbtcdjE2yUP-kX5qa0fPVxNTZGAkxE7jxRgW25M8HJ1HYw9EIBZVOkgXn9BUMpZ9bqHbo7HiqIsMfeNx1hk8VUBCxwidZAf3onGjM65LpEH-Nly41gaXxzOe_4seLShiApA5wjKAxLlA0oBbJ2yqlQVrvkCR092YaJmbS0y-DqC6WudLqkItWAIF5nBIkdUPZq2exXsqCmR0UXA03e4akcVYM.A-ouhPXBTrbB0Q15QTrmUw&id_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJodHRwczovL3d3dy5taWxlemVyby5jb20vbWlsZXplcm8tc3RhZ2UvYXBwTmFtZSI6Ik1pbGVWaXNpb24iLCJhdXRob3JpemF0aW9uIjp7Imdyb3VwcyI6WyJQcmlvcml0eUV4cHJlc3MiLCJQcmlvcml0eUV4cHJlc3MgTWlsZVZpc2lvbiJdLCJyb2xlcyI6WyJNaWxlVmlzaW9uX1VzZXIiXSwicGVybWlzc2lvbnMiOltdfSwib3JnYW5pemF0aW9uIjp7Im9yZ05hbWUiOiI3NzE3Y2Q5ZC0zNzFiLTRlMWItYjEwNS1iMjlkMWFjYmRkZjciLCJvcmdJZCI6Ijc2YzIzNjg3LTQzYzQtNGZlMy1iNWYwLTRjZGQzODZlNzg5NSIsIkFsYWJvIjp7Im9yZ05hbWUiOiI3NzE3Y2Q5ZC0zNzFiLTRlMWItYjEwNS1iMjlkMWFjYmRkZjciLCJvcmdJZCI6Ijc2YzIzNjg3LTQzYzQtNGZlMy1iNWYwLTRjZGQzODZlNzg5NSJ9fSwib2JqZWN0cyI6eyJkcml2ZXJJZCI6Ijc0MmMwMGNlLWZiYzMtNDM3MC1iN2IxLTY3YWZhMTUwZTBkOSJ9LCJpc3MiOiJodHRwczovL21pbGV6ZXJvLXN0YWdlLmF1dGgwLmNvbS8iLCJzdWIiOiJhdXRoMHw1ZGQ2NTA3ZmY4M2VlNjBmMjM5M2VkMjAiLCJhdWQiOiJ6Z08xVVF2UmZQajJOQ2pvbWFlYVFTSEpBYzJWOWRmMyIsImlhdCI6MTY4NjU5NDk2NywiZXhwIjoxNjg2NjMwOTY3fQ.SqS0PeAFSORexIVIfP0jss2ZvqvMo70yRE7Q76lN9hU&scope=openid%20authorization%20organization%20objects&expires_in=86400&token_type=Bearer&state=fo4HDKKwP8mVam3_VFt9P-PSfUN6BnS3');
```

Comment:
The token can be generated from another independent function and saved, then called here by its defined parameter every single time when needed to generate a token to acces.